test/infra/cchost.py

```diff
@@ -192,6 +192,10 @@ async def _start_service(self):
                 raise
 
     async def _start_process(self):
+        # Ensure SGX_AESM_ADDR is not set when starting cchost.
+        cchost_env = os.environ.copy()
+        cchost_env.pop("SGX_AESM_ADDR", None)
```

Comment:
```suggestion
        cchost_env.pop("SGX_AESM_ADDR")
```

Reply:
Without a default set (here `None`) then `pop` will fail if SGX_AESM_ADDR is already unset (or was never set)

Reply:
Oh my bad, I somehow assumed `None` was already the default value (I think I mixed it up with the `get` method).